nmoscommon/auth/nmos_auth.py

```diff
@@ -120,11 +120,16 @@ def getPublicKey(self):
         return pubKey
 
     def processAccessToken(self, auth_string):
-        token_type, token_string = auth_string.split(None, 1)
+        # Auth string is of type 'Bearer xAgy65..'
+        if auth_string.find(' ') > -1:
```

Comment:
Would it be worth using the following to be a little clearer what it's searching for?
```suggestion
        if auth_string.find('Bearer ') == 1:
```

Reply:
I see lower down why it's done that way. Personally I'd find it less confusing if the split() was carried out at the top, then if len() == 1 it's just a token, but if len() == 2 and the first part is 'Bearer' it's the other format. It clearly works either way though and there are comments, so if you don't agree it's fine as is.

Reply:
I think with the comments it should be fine. Otherwise you start having to index into lists which the has the issue of introducing 'magic numbers'